### PR TITLE
4.1.0 build

### PR DESCRIPTION
**Pull request by psteyer**
Building zeep for customer request

JIRA ticket: https://anaconda.atlassian.net/browse/PKG-165

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,14 +14,16 @@ source:
...
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  skip: True  # [py<36]
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Should be `setuptools >=40.6.0`, see https://github.com/mvantellingen/python-zeep/blob/1e55eeb6b0b992e90234296b1a1392253528dfef/pyproject.toml#L2

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,13 +37,19 @@ requirements:
...
+  requires:
+    - pip
 
 about:
   home: http://www.python-zeep.org
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: A fast and modern Python SOAP client
+  doc_url: http://docs.python-zeep.org/
```
Comment (varlackc) [inline recipe/meta.yaml]: We might want to remove the duplicate `doc_urls` and `dev_urls`

https://github.com/AnacondaRecipes/zeep-feedstock/blob/dea42e77e27a83237f65e5bb3e92f2dfbd95d3b9/recipe/meta.yaml#L51&L66

File: recipe/meta.yaml [hunk 3]
```diff
@@ -35,13 +37,19 @@ requirements:
 test:
   imports:
     - zeep
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: http://www.python-zeep.org
```
Comment (varlackc) [inline recipe/meta.yaml]: Also we do seem to have some issues opening the `home url` and `doc_url` as an `https` we might want to specify in a comment that `https` is not supported for the given urls at this point.

**Review by varlackc — COMMENTED** [review]
We might want to take a look at the `urls`

File: recipe/meta.yaml (hunk 1 above)
Comment (psteyer) [reply recipe/meta.yaml]: added

File: recipe/meta.yaml (hunk 2 above)
Comment (psteyer) [reply recipe/meta.yaml]: Strange, I must of added those thinking there we not any. I went ahead and removed them

File: recipe/meta.yaml (hunk 3 above)
Comment (psteyer) [reply recipe/meta.yaml]: added the comments

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**